### DISCUSSION TITLE
Is Quasar still actively maintened?

### DISCUSSION THREAD

**Discussion by luispittagros** [quasarframework/quasar#15105] [Polls]
It feels wrong to complain because we are getting this great framework for free but I'm worried that with the current development speed, with no roadmap, in a year or two the development will almost come to a halt.

**Comment by metalsadman** [comment]
> It feels wrong to complain because we are getting this great framework for free but I'm worried that with the current development speed, with no roadmap, in a year or two the development will almost come to a halt.

https://quasar.dev/start/roadmap
https://github.com/quasarframework/quasar/blob/dev/ROADMAP.md

https://quasar.dev/start/release-notes
https://github.com/quasarframework/quasar/releases

Join discord to see git updates.
https://discord.com/channels/415874313728688138/423499262479958027
